bike_rent/models/product.py

```diff
@@ -14,3 +15,5 @@ class ProductProduct(models.Model):
 
     rent_ids = fields.One2many('bike.rent', 'bike_id', string='Rent Records')
     rent_duration = fields.Integer(string='Rent Duration (Days)')
+    product_bike_id = fields.Many2one('product.product', string='Bike ID', index=True, ondelete='cascade')
+    rented_bike_ids = fields.One2many('product.product', 'product_bike_id', string='Rented Bikes')
```

Comment:
why `rented`?

Reply:
Felt like I already had a few 'bike_id' fields throughout the module at that moment, so just wanted to be more precise with the name, e.g. these are bike ids specifically for rent purposes. Too much? Or is it the past tense? ;]